test/SlowTests/Issues/RavenDB-20780.cs

```diff
@@ -63,7 +63,7 @@ public void JsIndexStepsCountOverflowExceptionLimit()
                 
                 index.Execute(store);
                 
-                Indexes.WaitForIndexing(store);
+                Indexes.WaitForIndexing(store, allowErrors: true);
```

Comment:
isn't this still racy? Errors are written in the different tx than staleness. Maybe better to use WaitForIndexingErrors?

Reply:
Changed